### PR TITLE
Added resource name to process warnings

### PR DESCRIPTION
**Pull request by tabarra**
This was done to help see the culprit of random deprecation warnings.  
TODO: show the stack.

### CODE REVIEW

File: data/shared/citizen/scripting/v8/main.js [hunk 1]
```diff
@@ -161,6 +161,11 @@ const EXT_LOCALFUNCREF = 11;
 	const rawEmitter = new EventEmitter2();
 	const netSafeEventNames = new Set(['playerDropped', 'playerConnecting']);
 
+	//Process warnings
+	process.on('warning', (warning) => {
```
Comment (blattersturm) [inline data/shared/citizen/scripting/v8/main.js]: What if `process` isn't defined (e.g. clients)?

File: data/shared/citizen/scripting/v8/main.js [hunk 2]
```diff
@@ -297,6 +302,7 @@ const EXT_LOCALFUNCREF = 11;
 		return frames;
 	}
 	
+	Error.stackTraceLimit = 16;
```
Comment (blattersturm) [inline data/shared/citizen/scripting/v8/main.js]: is this related to the PR's purpose?

Comment (tabarra) [reply data/shared/citizen/scripting/v8/main.js]: The default is 10.  
Not enough in many cases.

**Comment by wilywork** [comment]
i add in my server, but not work.

**Comment by jaymo1011** [comment]
> i add in my server, how do I test?

What? The changes in this PR haven't been approved, cause at least one breaking issue and the requested changes haven't been made. How are you using them and why are you using them?

**Comment by wilywork** [comment]
> > i add in my server, how do I test?
> 
> What? The changes in this PR haven't been approved, cause at least one breaking issue and the requested changes haven't been made. How are you using them and why are you using them?

I added the lines to my server's file, I found it very useful and decided to use it to identify bottleneck problems on my server, but I didn't receive any messages from this event. my server keeps firing: "server thread hitch warning: timer interval of 477 milliseconds" and I can't identify who is causing it. I thought this change would help me.

**Comment by tabarra** [comment]
@wilywork This is not what this PR do, it is for node process warnings like deprecation warnings.
